marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/artifacts/getArtifactsWithProjectPaths.sjs

```diff
@@ -31,7 +31,10 @@ const userArtifactQuery = cts.andQuery([
     consts.FLOW_COLLECTION,
     consts.ENTITY_MODEL_COLLECTION,
     "http://marklogic.com/data-hub/steps/ingestion",
-    "http://marklogic.com/data-hub/steps/mapping"
+    "http://marklogic.com/data-hub/steps/mapping",
```

Comment:
This can just be the steps collection, since we're now supporting all of them.

Reply:
Changed to use the STEP_COLLECTION constant.